src/pages/workspaces/workspace/analysis/modals/DeleteDiskChoices.test.ts

```diff
@@ -0,0 +1,293 @@
+      const setDeleteDiskSelected = jest.fn()
+
+      // Act
+      render(h(DeleteDiskChoices, {
+        persistentDiskCost: pdCost,
+        toolLabel: 'RStudio',
+        cloudService: cloudServices.GCE,
+        deleteDiskSelected,
+        setDeleteDiskSelected
+      }))
+
+      // Assert
+      expect(screen.getByText('move them to the workspace bucket.')
+        .closest('a'))
+        .toHaveAttribute('aria-label', 'RStudio save help')
+    })
+
+    it('Should show safefileshelp option for jupyter', () => {
+      // Arrange
+      const pdCost = formatUSD(1.01)
+      const deleteDiskSelected = false
+      const setDeleteDiskSelected = jest.fn()
+
+      // Act
+      render(h(DeleteDiskChoices, {
+        persistentDiskCost: pdCost,
+        toolLabel: 'Jupyter',
+        cloudService: cloudServices.GCE,
+        deleteDiskSelected,
+        setDeleteDiskSelected
+      }))
+
+      // Assert
+      expect(screen.getByText('move them to the workspace bucket.')
+        .closest('a'))
+        .toHaveAttribute('aria-label', 'Save file help')
+    })
+
+    it('Should show safefileshelp option for azure', () => {
+      // Arrange
+      const pdCost = formatUSD(1.01)
+      const deleteDiskSelected = false
+      const setDeleteDiskSelected = jest.fn()
+
+      // Act
+      render(h(DeleteDiskChoices, {
+        persistentDiskCost: pdCost,
+        toolLabel: 'Jupyter',
+        cloudService: cloudServices.AZURE,
+        deleteDiskSelected,
+        setDeleteDiskSelected
+      }))
+
+      // Assert
+      expect(screen.getByText('move them to the workspace bucket.')
+        .closest('a'))
+        .toHaveAttribute('href', 'https://support.terra.bio/hc/en-us/articles/12043575737883')
+    })
+  })
+
+  describe('DeleteEnvironment', () => {
+    it('Should properly render when provided no disk/runtime', () => {
+      // Arrange
+      const setDeleteDiskSelected = jest.fn()
+      const setViewMode = jest.fn()
+
+      // Act
+      render(h(DeleteEnvironment, {
+        id: 'not-relevant',
+        deleteDiskSelected: false,
+        setDeleteDiskSelected,
+        setViewMode,
+        renderActionButton,
+        hideCloseButton: false,
+        onDismiss: () => {},
+        toolLabel: 'RStudio'
+      }))
+
+      // Assert
+      screen.getByText('Deleting your application configuration and cloud compute profile will also')
+    })
+    it.each([
+      { disk: getDisk() },
+      { disk: getAzureDisk() },
+    ])('Should properly render when provided no runtime but a disk', ({ disk }) => {
+      // Arrange
+      const setDeleteDiskSelected = jest.fn()
+      const setViewMode = jest.fn()
+
+      // Act
+      render(h(DeleteEnvironment, {
+        id: 'not-relevant',
+        deleteDiskSelected: false,
+        persistentDisk: disk,
+        setDeleteDiskSelected,
+        setViewMode,
+        renderActionButton,
+        hideCloseButton: false,
+        onDismiss: () => {},
+        toolLabel: 'RStudio'
```

Comment:
Nit: Azure currently doesn't have RStudio for a tool. 
I think there's a missing code coverage point here as well - consider putting toolLabel in the it.each. Use RStudio for the "GCP" test and "JupyterLab" for the Azure test.

You can additionally add cloudPlatform into the it.each list and print it in the test string - this way the tests can be more easily identified and differentiated.